### ISSUE TITLE
Specifying what TD terms are specified by different binding templates

### ISSUE THREAD

**Issue by egekorkan** [w3c/wot-binding-templates#218]
Currently, we specify what each binding template needs to provide. However, we are not putting the relationships to the TD terms clearly. There are some explanations at https://w3c.github.io/wot-architecture/#sec-hypermedia-controls but we can be more explicit here. From initial thinking, the relationships should be:
- Protocols: Everything inside a form can be specified but the `contentType` is not necessarily part of it.
- Payloads: Everything inside the form except header and protocol specific terms, and data schema
- Platforms: Combination of the two above

**Comment by egekorkan** [comment]
I propose to postpone after #223
